### PR TITLE
Jack Lab 9, v2 complete

### PR DESCRIPTION
**Pull request by jackamoss2**

### CODE REVIEW

**Comment by jackamoss2** [comment]
turning in lab 9 version 1

**Comment by jackamoss2** [comment]
turning in lab 9 version 2

File: code/jack/python/lab09.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+class ATM():
+    def __init__(self):
+        self.balance = 0
+        self.interest = 0.001
+        self.transactions = []
+
+    def check_balance(self):
+        return self.balance
+    
+    def deposit(self, amount, interest = False):
```
Comment (brucestull) [inline code/jack/python/lab09.py:12]: * Great use of a kwarg to specify if the deposit is interest or not.

File: code/jack/python/lab09.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
+# Lab 9: ATM machine
+
```
Comment (brucestull) [inline code/jack/python/lab09.py:2]: * Great code layout/formatting.
* Readable code.

**Review by brucestull — APPROVED** [review]
[Lab 9: ATM](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/09%20ATM.md)

**Completed**

**Pull request merged**